source/calculus/source/01-LT/02.ptx

```diff
@@ -341,13 +377,26 @@ the function as <m>x</m> tends to 2?
             <li><p> The limit does not exist because you are dividing by zero when <m>x = 0</m> for <m>f(x).</m></p></li>
 
         </ol>
-         </statement> </task>   
+         </statement> 
+         <answer>
+          <p>
+            C. The limit does not exist because the function is oscillating between -1 and 1.
+          </p>
+         </answer>
+         </task>   
  <task> <statement>
         <p>Would your conclusion that resulted from <xref ref = "activity-limits-numerically5"/> change if the function was <m>f(x) = \cos(1/x)</m> or <m>f(x) = \tan(1/x)</m>?</p>
-        </statement> </task>   
+        </statement>
+        <answer>
+          <p>
+            Yes, the limit of  <m>f(x) = \tan(1/x)</m> would be 0 as <m> x </m> tends to 0 however <m>f(x) = \cos(1/x)</m> oscillates as <m> x </m> tends to 0 . 
```

Comment:
Both limits still don't exists. I think you were thinking of x tending to infinity for the tan one.